conans/client/installer.py

```diff
@@ -200,7 +199,6 @@ def build_package(self, node, keep_build, recorder, remotes):
 
         # BUILD & PACKAGE
         with package_layout.conanfile_read_lock(self._output):
-            _remove_folder_raising(package_folder)
```

Comment:
This is the only possible minor functional change. The ``package_folder`` was removed in this point when building from source. Now it is removed earlier in the process, as soon as the build will start.